livre/core/render/GLContext.h

```diff
@@ -33,7 +33,7 @@ namespace livre
 class GLContext
 {
 public:
-    LIVRECORE_API GLContext();
+    LIVRECORE_API GLContext( const GLEWContext* glewContext );
```

Comment:
It depends on the system. This bug was that glCreateProgram crashes on OS X/X11/64 bit, unless you do proper extension handling.